### ISSUE TITLE
contact diary: use already existing code on github

### ISSUE THREAD

**Issue by ouboub** [corona-warn-app/cwa-wishlist#339]
It seems to me that this subject is worth its own issue. I just learnt that the app kontakt-tagebuch was removed from the App story. I really don't want to comment on this (unwise) decision by apple to remove a free app that helps to fight the pandemic, I merely want to point out that the apps' developer, Stefan Traut, generously has made his code public under the MIT license on github [download link](https://github.com/funkenstrahlen/contact-journal.git) could that code (save its access to the contact list) included in the Corona Warn app. I, unfortunately, don't know enough swift to judge.

regards

Uwe Brauer

**Comment by Ein-Tim** [comment]
Just to ensure that I understand you correctly:
You want to add the whole "Kontakt-Tagebuch"-app go get added to CWA? 
This is not that simple as it sounds, the integration maybe would not be the big part, but the UI would have to be adapted to the CWA UI.

Leave this Issue open since this is the wishlist, but this seems very unrealistic to me. 
I think it would be better if you open new Issues which requests features from this App. (Take a look at #310) 👍

**Comment by ouboub** [comment]
>>> "ET" == Ein Tim <notifications@github.com> writes:

> Just to ensure that I understand you correctly:


> You want to add the whole "Kontakt-Tagebuch"-app go get added to CWA? 

yes, save the contact list feature which is controversial as I understand.

> This is not that simple as it sounds, the integration maybe would not
> be the big part, but the UI would have to be adapted to the CWA UI.

Maybe one could link both  apps? 

For example if you export information of a specific app, the share
functionality allows you either to use Apple's own apps like mail, but
although 3rd party apps. So,  maybe for this specific purpose, that would be
possible to, so the button, add contacts would open that app?

> Leave this Issue open since this is the wishlist, but this seems very
> unrealistic to me.

> I think it would be better if you open new Issues which requests
> features from this App. (Take a look at #310) 👍

I did, it looks a bit like reinvent the wheel again.
